error/src/lib.rs

```diff
@@ -560,6 +542,68 @@ impl CaliptraError {
     pub const ROM_KAT_LMS_DIGEST_MISMATCH: CaliptraError = CaliptraError::new_const(0x90070002);
 
     pub const ROM_INTEGRITY_FAILURE: CaliptraError = CaliptraError::new_const(0x90080001);
+
+    /// CFI Errors
+    pub const CFI_PANIC_UNKNOWN: CaliptraError = CaliptraError::new_const(0xA0000001);
+    pub const CFI_PANIC_COUNTER_CORRUPT: CaliptraError = CaliptraError::new_const(0xA0000002);
+    pub const CFI_PANIC_COUNTER_OVERFLOW: CaliptraError = CaliptraError::new_const(0xA0000003);
+    pub const CFI_PANIC_COUNTER_UNDERFLOW: CaliptraError = CaliptraError::new_const(0xA0000004);
+    pub const CFI_PANIC_COUNTER_MISMATCH: CaliptraError = CaliptraError::new_const(0xA0000005);
+    pub const CFI_PANIC_ASSERT_EQ_FAILURE: CaliptraError = CaliptraError::new_const(0xA0000006);
+    pub const CFI_PANIC_ASSERT_NE_FAILURE: CaliptraError = CaliptraError::new_const(0xA0000007);
+    pub const CFI_PANIC_ASSERT_GT_FAILURE: CaliptraError = CaliptraError::new_const(0xA0000008);
+    pub const CFI_PANIC_ASSERT_LT_FAILURE: CaliptraError = CaliptraError::new_const(0xA0000009);
+    pub const CFI_PANIC_ASSERT_GE_FAILURE: CaliptraError = CaliptraError::new_const(0xA000000A);
+    pub const CFI_PANIC_ASSERT_LE_FAILURE: CaliptraError = CaliptraError::new_const(0xA000000B);
+    pub const CFI_PANIC_TRNG_FAILURE: CaliptraError = CaliptraError::new_const(0xA000000C);
+    pub const CFI_PANIC_UNEXPECTED_MATCH_BRANCH: CaliptraError =
+        CaliptraError::new_const(0xA000000D);
+    pub const CFI_PANIC_FAKE_TRNG_USED_WITH_DEBUG_LOCK: CaliptraError =
+        CaliptraError::new_const(0xA000000E);
+}
+
+impl From<caliptra_cfi_lib::CfiPanicInfo> for crate::CaliptraError {
+    fn from(val: caliptra_cfi_lib::CfiPanicInfo) -> Self {
+        match val {
```

Comment:
I thought we were going to try and get rid of this remapping logic and try to make the error codes consistent.

Reply:
Yeah that was the plan initially, but I think it's better this way since each caller of the library could define their own conversion. If we have another platform that integrates DPE which also uses CFI, it's error codes would be different from the Caliptra CFI error codes.

Reply:
Doing the conversion requires a significant lookup table, which it'd be nice to avoid, especially since this is all in the same firmware bundle. Also, the branchy nature of that code is best to avoid in a CFI error-handling context.

Reply:
Reworked this to simply add the CfiPanicInfo offset to a CaliptraError base constant so the branching should be eliminated.